homeassistant/helpers/update_coordinator.py

```diff
@@ -250,9 +261,17 @@ def _schedule_refresh(self) -> None:
         hass = self.hass
         loop = hass.loop
 
-        next_refresh = (
-            int(loop.time()) + self._microsecond + self._update_interval_seconds
-        )
+        update_interval = self._update_interval_seconds
+        if self._retry_after is not None:
+            self.logger.debug(
```

Comment:
I don't think we should have any logging statements in this method. We already log below when the exception has a retry after attribute.